SCAutolib/models/gui.py

```diff
@@ -0,0 +1,281 @@
+from SCAutolib import run
+
+import os
+from time import sleep, time
+import cv2
+import numpy as np
+import pandas as pd
+from io import StringIO
+import pytesseract
+import keyboard
+import uinput
+import functools
+
+
+class Screen():
+    """Captures the screenshots"""
+
+    def __init__(self, directory: str):
+        self.directory = directory
+        self.screenshot_num = 1
+
+    def screenshot(self):
+        """
+        Runs ffmpeg to take a screenshot.
+
+        The filename of screenshot is then returned.
+        """
+
+        filename = f'/tmp/SC-tests/{self.screenshot_num}.png'
+        run(['ffmpeg', '-hide_banner', '-y', '-f', 'kmsgrab', '-i', '-', '-vf',
```

Comment:
ffmpeg output could be redirected or suppressed to make logs shorter and cleaner, however, there can be some log about taken screenshot.